src/test/java/hw8/pages/custom/MultiDropdown.java

```diff
@@ -0,0 +1,252 @@
+package hw8.pages.custom;
+
+import com.epam.jdi.light.asserts.generic.UISelectAssert;
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.common.TextTypes;
+import com.epam.jdi.light.elements.base.JDIBase;
+import com.epam.jdi.light.elements.base.UIListBase;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.interfaces.base.HasLabel;
+import com.epam.jdi.light.elements.interfaces.base.ICoreElement;
+import org.openqa.selenium.By;
+
+import java.util.List;
+
+import static com.epam.jdi.light.driver.WebDriverByUtils.fillByTemplate;
+import static com.epam.jdi.light.elements.init.UIFactory.$;
+import static com.epam.jdi.light.elements.init.UIFactory.$$;
+import static com.epam.jdi.light.logger.LogLevels.DEBUG;
+import static com.epam.jdi.tools.EnumUtils.getEnumValues;
+import static com.epam.jdi.tools.LinqUtils.*;
+import static java.util.Arrays.asList;
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
+public class MultiDropdown extends UIListBase<UISelectAssert<?,?>>
```

Comment:
Do you really need to implement your own element from UIListBase ?
Maybe, it will be easier to override some methods from DropdownExpand ?

Reply:
I've changed that type, there was no need to use a custom class for this field.